server/services/twitter.js

```diff
@@ -0,0 +1,62 @@
+import superagent from 'superagent';
+import { createTweet } from '../model/tweet';
+
+async function getTwitterAccessToken() {
+  // TODO: reimplement after testing
+  const urlEncodedK = encodeURIComponent('qpsVJlsTg22FFKr1fwzkzU4Ew');
+  const urlEncodedS = encodeURIComponent('kmXPnluTYkl7oH4mRlC1EgthH8KLT9mBpezgK3wBaFQviC23Ru');
```

Comment:
Yeah, we're OK

![screen shot 2017-06-05 at 11 36 16](https://cloud.githubusercontent.com/assets/1394592/26781125/e45d2d24-49e4-11e7-8994-71cf246f213c.png)